### PR TITLE
Add label image conversion

### PR DESCRIPTION
**Pull request by keigohtr**
## What is this PR for?

Our precondition is that the format of label image is RGBA. Since we converted label images from RGBA to RGB automatically via pillow, I decided to specify how to convert it manually.

## This PR includes

- Add alpha_to_color conversion

## What type of PR is it?

Minor

## What is the issue?

N/A

## How should this be tested?

N/A

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -207,6 +208,20 @@ def log_every(self, iterable, print_freq, header=None):
         print('{} Total time: {}'.format(header, total_time_str))
 
 
+def worker_init_fn(worker_id):
+    random.seed(worker_id)
+
+
+def alpha_to_color(image, color=(0, 0, 0)):
+    x = np.array(image)
+    r, g, b, a = np.rollaxis(x, axis=-1)
```
Comment (ysku) [inline utils.py:217]: understand that image here is always RGBA !!

File: utils.py [hunk 2]
```diff
@@ -207,6 +208,20 @@ def log_every(self, iterable, print_freq, header=None):
...
+    r[a == 0] = color[0]
+    g[a == 0] = color[1]
+    b[a == 0] = color[2]
```
Comment (ysku) [inline utils.py:220]: it was difficult to get it...

**Review by ysku — APPROVED** [review]
Looks Good To Me

File: utils.py (hunk 2 above)
Comment (keigohtr) [reply utils.py:220]: numpy magic

**Comment by keigohtr** [comment]
Close this PR since pillow's RGBA to RGB does the almost same thing (and we don't need to set the specific RGB value according to A plane).
I will create another PR for setting the random seed.

**Pull request closed without merging**